specs/ClearBrowsingData.md

```diff
@@ -0,0 +1,233 @@
+        * Isolated Origins
+        * Trust Tokens
+        * Conversions Data
+    * Http Cache 
+    * Download History
+    * General Autofill
+    * Password Autofill
+    * Bookmarks
+    * Settings
+    * Content Settings
+    * Local Custom Dictionary
+
+
+# Examples
+
+## Win32 C++
+```cpp
+/// Function to clear the password form data 
+bool EnvironmentComponent::ClearPasswordAutofillData() 
+{
+    wil::com_ptr<ICoreWebView2Environment5> environment;
+    webView->get_Environment(&environment);
+    CHECK_FAILURE(environment->ClearBrowsingData(COREWEBVIEW2_BROWSING_DATA_KIND_PASSWORD_AUTOFILL,
+        Callback<ICoreWebView2ClearBrowsingDataHandler>(
+            [this](HRESULT error, bool success) -> HRESULT {
+                if (success){
+                    return true;
+                } else {
+                    return false;
+                }
+                return S_OK;
+            })
+            .Get()));
+}
+```
+
+### .NET, WinRT
+```c#
+webView2Control.NavigationStarting += ClearPasswordAutofillData;
+
+private void ClearPasswordAutofillData() 
+{
+    var environment = webView2Control.CoreWebView2.Environment;
+    try
+    {
+        await environment.ClearBrowsingDataAsync(COREWEBVIEW2_BROWSING_DATA_KIND_PASSWORD_AUTOFILL); 
+    }
+    catch (System.Runtime.InteropServices.COMException exception)
+    {
+        // An exception occured
+    }
+}
+
+```
+
+# API Notes
+See [API Details](#api-details) section below for API reference.
+
+
+# API Details
+
+## Win32 C++
+
+```IDL
+interface ICoreWebView2Environment5
+interface ICoreWebView2ClearBrowsingDataCompletedHandler;
+
+/// Specifies the datatype for the 
+/// `ICoreWebView2Environment5::ClearBrowsingData` method.
+[v1_enum]
+typedef enum COREWEBVIEW2_BROWSING_DATA_KIND {
+  /// Specifies app cache data.
+  COREWEBVIEW2_BROWSING_DATA_KIND_APP_CACHE = 0X01,
+
+  /// Specifies file systems data.
+  COREWEBVIEW2_BROWSING_DATA_KIND_FILE_SYSTEMS = 0X02,
+
+  /// Specifies indexeddb data.
+  COREWEBVIEW2_BROWSING_DATA_KIND_INDEXEDDB = 0X03,
+
+  /// Specifies local storage data.
+  COREWEBVIEW2_BROWSING_DATA_KIND_LOCAL_STORAGE = 0X04,
+
+  /// Specifies web SQL data.
+  COREWEBVIEW2_BROWSING_DATA_KIND_WEB_SQL = 0X05,
+
+  /// Specifies cache storage.
+  COREWEBVIEW2_BROWSING_DATA_KIND_CACHE_STORAGE = 0X06,
+
+  /// Specifies dom storage data. This browsing data kind is inclusive 
+  /// of COREWEBVIEW2_BROWSING_DATA_KIND_APP_CACHE, COREWEBVIEW2_BROWSING_DATA_KIND_FILE_SYSTEMS,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_INDEXEDDB, COREWEBVIEW2_BROWSING_DATA_KIND_LOCAL_STORAGE,
+  /// COREWEBVIEW2_BROWSING_DATA_KIND_WEB_SQL, COREWEBVIEW2_BROWSING_DATA_KIND_CACHE_STORAGE.
+  /// In addition to these data kinds, dom storage also includes embedder dom storage and
+  /// background fetch. 
+  COREWEBVIEW2_BROWSING_DATA_KIND_DOM_STORAGE = 0x07,
+  
+  /// Specifies http cookies data.
+  COREWEBVIEW2_BROWSING_DATA_KIND_COOKIES = 0x08,
+
+  /// Specifies media licenses.
```

Comment:
It would be good to elaborate here and cover what web platform feature we're talking about here. Are they downloaded by the media/video/audio element or something like that?